### PR TITLE
chore: cleanup docs

### PR DESCRIPTION
**Pull request by enjoyooor**
fixing sentence structures and minor issues in the docs

### CODE REVIEW

File: tachyon/base/numerics/README.md [hunk 1]
```diff
@@ -370,16 +369,14 @@ all types and `%`, `<<`, `>>`, `&`, `|`, `^` for integers) along with comparison
...
 as they eliminate redundant expressions that are unavoidable with the operator
-overloads. (Ideally the compiler should optimize those away, but better to avoid
+overloads. (Ideally, the compiler should optimize those away, but better to avoid
 them in the first place.)
 
 Type promotions are a slightly modified version of the [standard C/C++ numeric
-promotions
-](http://en.cppreference.com/w/cpp/language/implicit_conversion#Numeric_promotions)
-with the two differences being that *there is no default promotion to int*
+promotions](http://en.cppreference.com/w/cpp/language/implicit_conversion#Numeric_promotions) with the two differences being that *there is no default promotion to int*
```
Comment (chokobole) [inline tachyon/base/numerics/README.md:379]: Did you to this by mistake or intend? If this was a mistake, Could you change it back to original?

**Comment by chokobole** [comment]
Thanks for the contribution! :+1:

**Review by dongchangYoo — APPROVED** [review]
LGTM

**Review by Insun35 — APPROVED** [review]
LGTM

**Pull request merged**